lib/application.js

```diff
@@ -58,7 +58,10 @@ class BookkeepingApplication {
             if (monalisaConfig.enableSynchronization) {
                 const monALISASynchronizer = await this.createMonALISASynchronizer();
                 this.scheduledProcessesManager.schedule(
-                    () => monALISASynchronizer.synchronizeDataPassesFromMonALISA(),
+                    async () => {
+                        await monALISASynchronizer.synchronizeDataPassesFromMonALISA();
```

Comment:
I think we put capital letter only in the first words even for acronyms, so `monAlisa`